airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -245,13 +248,16 @@ class AverageSessionLength(IncrementalAmplitudeStream):
 
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         response_data = response.json().get(self.data_field, [])
-        if response_data:
+        if response_data and response_data["series"]:
```

Comment:
is the key `series` guaranteed in response?

Reply:
I have only added checks to catch errors I found while running sync - would be too many checks otherwise. I never got KeyError for series while testing.

Reply:
we can confirm from the API doc if `series` is guaranteed or not.
If not then we can do something like `response_data.get('series')`

Reply:
I suspect it is guranteed.

Reply:
I think so too. Let me check anyway

Reply:
It is guaranteed (Get average session length part of https://www.docs.developers.amplitude.com/analytics/apis/dashboard-rest-api/)

Reply:
Should we still add a check if series is present in the response instead of taking it for granted?

Reply:
I'm checking if response_data["series"] is not empty. I'm assuming key is present..

Reply:
Okay.